src/main/webapp/WEB-INF/views/cars/createCar.jsp

```diff
@@ -0,0 +1,18 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+  <title>Create car</title>
```

Comment:
common mistakes
it is bad practice to call "create", it is better to rename "add"
check all homework